CHANGELOG.md

```diff
@@ -25,7 +25,7 @@ Bugfix release
 ```shell-session
 $ git shortlog -sn v1.0.1...
      7  dependabot[bot]
-     5  John Reese
+     5  John Reesemk
```

Comment:
lolwat?

Reply:
`mk` is definitely not my password anywhere 👀